experiment/run_experiment.py

```diff
@@ -173,8 +169,6 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
             Requirement(not local_experiment, str, True, ''),
         'worker_pool_name':
             Requirement(not local_experiment, str, False, ''),
-        'experiment':
-            Requirement(False, str, False, ''),
```

Comment:
This is the experiment name, in case anyone set it in the config.
It's probably OK to remove it as well, as we expect users to pass the name from the cmdline?

Reply:
Yeah it should be set from the command line since it changes very often (literally every use)